src/responder/common/cache_req/cache_req.c

```diff
@@ -1114,7 +1114,8 @@ struct tevent_req *cache_req_send(TALLOC_CTX *mem_ctx,
     }
     state->first_iteration = true;
 
-    CACHE_REQ_DEBUG(SSSDBG_TRACE_FUNC, cr, "New request '%s'\n", cr->reqname);
+    CACHE_REQ_DEBUG(SSSDBG_TRACE_FUNC, cr, "New request [CID #%u] '%s'\n",
+                    rctx->client_id_num, cr->reqname);
```

Comment:
Hi, that's a good idea. Were you thinking of something like:
~~~
[cache_req_send] (0x0400): CR #2: New request [CID #1] 'Initgroups by name'
[cache_req_send] (0x0400): REQ_TRACE: [CR #2] 'Initgroups by name' linked to [CID #1] '
~~~

Or combine them all into the same single log message? Here I will use `REQ_TRACE` as an identifier which can filter the linking relationship log messages (other will be in the backend)

Reply:
Hi,

my suggestion was more about the source code, so that it is easy to see that it is not an ordinary debug message. But adding a keyword to the log messages might be useful as well.

bye,
Sumit

Reply:
Fixed, see this updated commit and https://github.com/SSSD/sssd/pull/5620/commits/46023c0b5abd995b295e9b47e13fc8b642372a8c